clients/pkg/promtail/promtail.go

```diff
@@ -119,7 +120,7 @@ func (p *Promtail) reloadConfig(cfg *config.Config) error {
 		return errConfigNotChange
 	}
 	newConf := cfg.String()
-	level.Info(p.logger).Log("msg", "Reloading configuration file", "newConf", newConf)
+	level.Info(p.logger).Log("msg", "Reloading configuration file", "newConf hash", xxhash.Sum64String(newConf))
```

Comment:
Why did you use xxhash here? Wouldn't md5 be easier for a user to check against?

Reply:
👌,you are right